app/presenters/feed_entry_presenter.rb

```diff
@@ -18,7 +18,10 @@ def atom_url
   end
 
   def updated
-    Time.zone.parse(result["public_timestamp"])
+    # Collections now uses London time. Convert it back to it's former zone
+    # of UTC in the domain of atom feeds to ensure backwards compatibility.
+
+    Time.find_zone("UTC").parse(result["public_timestamp"])
```

Comment:
sorry @hannako - I think we need `.iso8601` appended to this to bring it in line with the [iso standards](https://tools.ietf.org/html/rfc3339) for [date constructs](https://tools.ietf.org/html/rfc4287#section-3.3) in atom feeds.

Reply:
Thanks for Bevan.  

So I was following [this](https://validator.w3.org/feed/docs/atom.html#aboutThisDocument) which says all atom dates need to be in RFC3339.

We create the ID like so:

```
def id
   "#{url}##{updated.rfc3339}"
end
```

And [on the review app](https://govuk-collec-fix-time-z-aucqvo.herokuapp.com/government/organisations/hm-revenue-customs.atom) the updated field within the ID's, and the updated field itself are identical. 

Also, adding `.rfc3339` to `Time.find_zone("UTC").parse(result["public_timestamp"])` locally raises an error when the feed.atom.builder (tries to present it)[https://github.com/alphagov/collections/blob/9f339b6a8b78f43b5720aebd0e62c7af1e7bb7ea/app/views/feeds/feed.atom.builder#L8]. 

These two things lead me to suspect that Rails's Atom feed helpers do some magic to format the date as `.rfc3339`. Will see if I can find the source code to confirm.....

(Ah here it is)[https://github.com/rails/rails/blob/0a608bd987fde4f9bc5bcf4bcebdb181d199cf4f/actionview/lib/action_view/helpers/atom_feed_helper.rb#L170]

Reply:
👍 magical - thanks! Learned something for the day 😄